controller.go

```diff
@@ -227,6 +233,10 @@ func (c *controller) NetworkByID(id string) (Network, error) {
 	return nil, nil
 }
 
+func (c *controller) HandleHTTPRequest(w http.ResponseWriter, req *http.Request) {
```

Comment:
Same as above. Yes, there will be corresponding change to the Client. Since we don't have a client at this point, I feel it is the right time to get this done :)